### PR TITLE
Add new Bluesky logo, update X logo and rearrange PPHW services

### PR DESCRIPTION
**Pull request by gonuke**
Builds successfully on my local machine

### CODE REVIEW

File: services/icons/twitter-x.png
Comment (MicahGale) [inline services/icons/twitter-x.png:1]: A petty part of me doesn't want to acknowledge this name change.

**Review by MicahGale — APPROVED** [review]
LGTM.

**Review by lewisgross1296 — COMMENTED** [review]
Only thing, I'd want to be sure the bluesky logo doesn't render the checkered background.

I still have issues building the website on Elbrus, but if you built it and the checkered background is not present, I'm good to merge.

**Comment by gonuke** [comment]
I think the checkers are the way of rendering a transparent layer in the image

**Pull request merged**